ur10_moveit_config/package.xml

```diff
@@ -24,10 +24,12 @@
   <exec_depend>moveit_simple_controller_manager</exec_depend>
   <exec_depend>joint_state_publisher</exec_depend>
   <exec_depend>robot_state_publisher</exec_depend>
+  <exec_depend>tf2_ros</exec_depend>
   <exec_depend>xacro</exec_depend>
   <depend>ur_description</depend>
 
   <test_depend>roslaunch</test_depend>
   <buildtool_depend>catkin</buildtool_depend>
-  
+
+  <exec_depend>trac_ik_kinematics_plugin</exec_depend>
```

Comment:
Why is this down here?

Reply:
Because it was an addition to the dependencies the MSA generates.

Keeping things separate like this simplifies reviews when the configuration package gets updated.

Reply:
But wouldn't it be nice to make sure that all dependencies are listed and that list is complete? If this is correctly updated before the PR (which IMHO makes sense to expect), reviews should still be nice and clean, right?